src/org/zalando/stups/kio/api.clj

```diff
@@ -45,45 +45,42 @@
   [request]
   (clojure.walk/keywordize-keys (:tokeninfo request)))
 
-; shameless copy from essentials
-(defn require-special-uid
-  "Checks wether a given user is configured to be allowed to access this endpoint. Workaround for now."
-  [{:keys [configuration tokeninfo]}]
-  (let [allowed-uids (or (:allowed-uids configuration) "")
-        allowed (set (str/split allowed-uids #","))
-        uid (get tokeninfo "uid")]
-    (when (and (seq allowed-uids)
-               (not (allowed uid)))
-      (log/warn "ACCESS DENIED (unauthorized) because not a special user.")
-      (api/throw-error 403 "Unauthorized"))))
-
 (defn require-uid
   "Checks whether uid is present on token, throws 403 otherwise"
   [request]
   (when-not (from-token request "uid")
     (log/warn "ACCESS DENIED (unauthorized) because no uid in tokeninfo.")
     (api/throw-error 403 "Unauthorized")))
 
+(defn is-admin-in-realm?
+  [uid realm {:keys [configuration]}]
+  (when (and uid realm)
```

Comment:
So if uid or realm are missing this function returns nil. Since nil is falsy, that works, but for sake of clarity you can consider adding an explicit false as return value.